pdm.lock

```diff
@@ -6,7 +6,7 @@ groups = ["default", "docs", "email", "linting", "memray", "mypy", "testing", "t
 cross_platform = true
 static_urls = false
 lock_version = "4.3"
-content_hash = "sha256:6f68a60ed5be207473a66fe2368d57b082d703e44f85c1b1767e625d81519dbb"
+content_hash = "sha256:712240a1a52356c85f693200f95828b353fe8adcc480fb443c6294f20b4dbde0"
```

Comment:
why has this changed?

Reply:
I'm not used to PDM, but one of the CI workflows was failing: https://github.com/pydantic/pydantic/actions/runs/6639854334/job/18039074733